### PR TITLE
PayEventElem handle caller != beneficiary

### PR DESCRIPTION
**Pull request by peripheralist**
## What does this PR do and why?

Adds a case to PayEventElem to show both caller and beneficiary when they are not the same address.

- light refactoring to ActivityEventElem puts the `Subject` and caller/beneficiary on the same line, to allow aligning by baseline (needed because now in the case where caller != beneficiary, right side text uses a smaller font size)
- adds a "title" property to FormattedAddress
- adds missing "caller" key to a few project activity events, so we can show on all events

## Screenshots or screen recordings

image
image
image

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

File: src/components/activityEventElems/ActivityElement/ActivityElement.tsx [hunk 1]
```diff
@@ -9,16 +10,36 @@ import { contentLineHeight, smallHeaderStyle } from '../styles'
...
+
+  return beneficiary &&
+    caller &&
+    beneficiary.toLowerCase() !== caller.toLowerCase() ? (
     <div
       style={{
-        display: 'flex',
-        justifyContent: 'space-between',
-        alignContent: 'space-between',
+        lineHeight: contentLineHeight,
```
Comment (wraeth-eth) [inline src/components/activityEventElems/ActivityElement/ActivityElement.tsx:27]: Any chance we can get rid of this?

**Review by wraeth-eth — CHANGES_REQUESTED** [review]

File: src/components/activityEventElems/ActivityElement/ActivityElement.tsx (hunk 1 above)
Comment (peripheralist) [reply src/components/activityEventElems/ActivityElement/ActivityElement.tsx:27]: actuallyyy i think with the change i just made this is unnecessary now. will verify

Comment (peripheralist) [reply src/components/activityEventElems/ActivityElement/ActivityElement.tsx]: done

**Review by tomquirk — APPROVED** [review]
lgtm @peripheralist nice addition!

**Pull request merged**